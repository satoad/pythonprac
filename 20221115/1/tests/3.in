@objcount
class C:
    def __init__(self, a):
        self.a = str(a)

    def __str__(self):
        return self.a + ' abobs'

c, d, e = C(5), C(10), C(15)
print(C.counter)
print(c, d, e)
del c
print(C.counter)
del e
print(C.counter)