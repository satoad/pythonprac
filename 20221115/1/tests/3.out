3
5 abobs 10 abobs 15 abobs
2
1
